### DISCUSSION TITLE
v4.4.1

### DISCUSSION THREAD

**Discussion by gyc123456-1** [gyc123456-1/pynotepad#11] [Announcements]
+新增`detect_encoding`函数用来检测编码
~更改自动检测编码的逻辑

<hr /><em>This discussion was created from the release <a href='https://github.com/gyc123456-1/pynotepad/releases/tag/v4.4.1'>v4.4.1</a>.</em>

**Comment by MicGan6** [comment]
准备开始适配了（doge）
